schema/snapcraft.json

```diff
@@ -1053,6 +1053,7 @@
                             "items": {
                                 "type": "string",
                                 "enum": [
+                                    "core22-lifecycle",
```

Comment:
This name suggests that the lifecycle will be exactly the same as core22, and it may not be exactly true depending on punctual changes we may decide to adopt. A name emphasizing this specific change could be safer (unless we use this switch later to run non-legacy code to process core20 projects, but new problems could appear in this scenario).